packages/odata-service-inquirer/src/types.ts

```diff
@@ -262,6 +262,11 @@ export type SystemSelectionPromptOptions = {
      *
      */
     defaultChoice?: string;
+    /**
+     * Only show the default choice in the system selection prompt, this is used to skip the system selection prompt if the default choice is already known.
+     * If the default choice is not known, the prompt will be shown as normal.
```

Comment:
```suggestion
     * If the `defaultChoice` value is not found in the systems choices, or the `defaultChoice` option is not specified ,the prompt will be shown as normal.
```

Reply:
Fixed in: af894387c3637e27cb722f67d25f72fcbc884114